histomicsui/web_client/dialogs/saveAnnotation.js

```diff
@@ -425,6 +444,15 @@ var SaveAnnotation = View.extend({
         return setValue;
     },
 
+    onRevert() {
+        if (this.annotation) {
+            delete this.annotation._meta;
+            delete this.annotation._styleFuncs;
+        }
+        this.annotation.trigger('revert:annotation', this.annotation, {});
```

Comment:
In girder we prefix our own events with `g:` to give them some modest namespace.  In some spots in HistomicsUI we prefix them with `h:`.  I think I'd prefer to change this to `h:revert:annotation`